### DISCUSSION TITLE
doris  查询慢

### DISCUSSION THREAD

**Discussion by xiaoL1211** [apache/doris#12430] [A - General / Q&A]
sql:
SELECT version,BITMAP_UNION_COUNT(count) as total FROM `version_day_uv` WHERE  (os_type = "windows") and  ((day_time >= "2022-08-30 00:00:00 +0800 CST") and (day_time <= "2022-09-08 00:00:00 +0800 CST")) and org_id >  20000  GROUP BY version;
![image](https://user-images.githubusercontent.com/5170688/188840769-5b48f5bf-1ae5-4f3f-b854-977df1eb7353.png)

当查询带org_id命中uv6，查询时间大概3-4秒
不带org_id命中uv5。查询1ms左右

org_id大概有5W个值
![image](https://user-images.githubusercontent.com/5170688/188841599-cbe2cb79-6d77-4dc9-a234-2481d7207759.png)

**Comment by dataalive** [comment]
能否发下完整的profile文件？

**Reply by xiaoL1211** [reply]
This table lists the latest 100 queries
Query:
    Summary:
          -  Query  ID:  2320cab9409147f6-95a3465b59a0b0f8
          -  Start  Time:  2022-09-07  09:36:26
          -  End  Time:  2022-09-07  09:36:30
          -  Total:  3s960ms
          -  Query  Type:  Query
          -  Query  State:  EOF
          -  Doris  Version:  1.1.0-rc05
          -  User:  root
          -  Default  Db:  default_cluster:ent
          -  Sql  Statement:  SELECT  version,BITMAP_UNION_COUNT(count)  as  total  FROM  `version_day_uv`  WHERE    (os_type  =  "windows")  and    ((day_time  >=  "2022-08-30  00:00:00  +0800  CST")  and  (day_time  <=  "2022-09-08  00:00:00  +0800  CST"))  and  org_id  >10  GROUP  BY  version
          -  Is  Cached:  No
        Execution  Summary:
              -  Analysis  Time:  2.335ms
              -  Plan  Time:  5.563ms
              -  Schedule  Time:  5.640ms
              -  Wait  and  Fetch  Result  Time:  3s946ms
    Execution  Profile  2320cab9409147f6-95a3465b59a0b0f8:(Active:  3s959ms,  %  non-child:  100.00%)
        Fragment  0:
            Instance  2320cab9409147f6-95a3465b59a0b0f9  (host=TNetworkAddress(hostname:100.109.66.188,  port:9060)):(Active:  3s942ms,  %  non-child:  0.00%)
                  -  FragmentCpuTime:  3s918ms
                  -  MemoryLimit:  2.00  GB
                  -  PeakMemoryUsage:  962.00  KB
                  -  PeakReservation:  0.00  
                  -  PeakUsedReservation:  0.00  
                  -  RowsProduced:  1
                BlockMgr:
                      -  BlockWritesOutstanding:  0
                      -  BlocksCreated:  0
                      -  BlocksRecycled:  0
                      -  BufferedPins:  0
                      -  BytesWritten:  0.00  
                      -  MaxBlockSize:  8.00  MB
                      -  TotalBufferWaitTime:  0ns
                      -  TotalEncryptionTime:  0ns
                      -  TotalIntegrityCheckTime:  0ns
                      -  TotalReadBlockTime:  0ns
                VDataBufferSender  (dst_fragment_instance_id=2320cab9409147f6--6a5cb9a4a65f4f07):
                      -  AppendBatchTime:  177.186us
                          -  ResultRendTime:  142.228us
                          -  TupleConvertTime:  24.297us
                      -  NumSentRows:  1
                VAGGREGATION_NODE  (id=1):(Active:  3s942ms,  %  non-child:  98.97%)
                      -  BuildTime:  3s902ms
                      -  ExecTime:  3s899ms
                      -  ExprTime:  53.15us
                      -  GetResultsTime:  393.834us
                      -  MergeTime:  0ns
                      -  PeakMemoryUsage:  36.00  KB
                      -  RowsReturned:  1
                      -  RowsReturnedRate:  0
                    VOLAP_SCAN_NODE  (id=0):(Active:  23.423ms,  %  non-child:  0.59%)
                          -  BatchQueueWaitTime:  22.888ms
                          -  BytesRead:  0.00  
                          -  NumDiskAccess:  1
                          -  NumScanners:  1
                          -  PeakMemoryUsage:  918.00  KB
                          -  RowsRead:  0
                          -  RowsReturned:  70.943K  (70943)
                          -  RowsReturnedRate:  3.028688M  /sec
                          -  ScannerBatchWaitTime:  84.844ms
                          -  ScannerSchedCount  :  5
                          -  ScannerWorkerWaitTime:  253.43us
                          -  TabletCount  :  1
                          -  TotalReadThroughput:  0.0  /sec
                        OlapScanner:
                              -  VconjunctExprTree:  null
                              -  BlockConvertTime:  0ns
                              -  BlockFetchTime:  0ns
                              -  ReaderInitTime:  1.195ms
                              -  RowsDelFiltered:  0
                              -  RowsPushedCondFiltered:  0
                              -  ScanCpuTime:  83.329ms
                              -  ScanTime:  0ns
                              -  ShowHintsTime_V1:  0ns
                            SegmentIterator:
                                  -  BitmapIndexFilterTimer:  0ns
                                  -  BlockLoadTime:  0ns
                                  -  BlockSeekCount:  0
                                  -  BlockSeekTime:  0ns
                                  -  BlocksLoad:  0
                                  -  CachedPagesNum:  0
                                  -  CompressedBytesRead:  1.63  MB
                                  -  DecompressorTimer:  0ns
                                  -  IOTimer:  0ns
                                  -  IndexLoadTime_V1:  0ns
                                  -  NumSegmentFiltered:  0
                                  -  NumSegmentTotal:  0
                                  -  RawRowsRead:  70.958K  (70958)
                                  -  RowsBitmapIndexFiltered:  0
                                  -  RowsBloomFilterFiltered:  0
                                  -  RowsConditionsFiltered:  0
                                  -  RowsKeyRangeFiltered:  0
                                  -  RowsStatsFiltered:  0
                                  -  RowsVectorPredFiltered:  0
                                  -  TotalPagesNum:  0
                                  -  UncompressedBytesRead:  0.00  
                                  -  VectorPredEvalTime:  0ns
